src/main/java/com/toptrumps/core/Attribute.java

```diff
@@ -1,4 +1,19 @@
 package com.toptrumps.core;
 
 public class Attribute {
+    private String attName;
+    private int attValue;
```

Comment:
@rjs68 you are already inside the context of Attribute :)
This is not an absolute requirement but you could've named theses properties just as ```name``` and ```value```